packages/lab/src/slider/RangeSlider.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  type ChangeEvent,
+  type HTMLAttributes,
+  type SyntheticEvent,
+  forwardRef,
+} from "react";
+
+import { useControlled, useFormFieldProps } from "@salt-ds/core";
+import clsx from "clsx";
+import { SliderThumb } from "./internal/SliderThumb";
+import { SliderTrack } from "./internal/SliderTrack";
+import { useSliderThumb } from "./internal/useSliderThumb";
+import { toFloat } from "./internal/utils";
+
+export interface RangeSliderProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, "onChange" | "defaultValue"> {
+  /**
+   * Initial value of the slider
+   */
+  defaultValue?: [number, number];
+  /**
+   * Disable the slider
+   */
+  disabled?: boolean;
+  /**
+   * A callback to format the display value in the tooltip
+   */
+  format?: (value: number) => string | number;
+  /**
+   * Position of the labels
+   */
+  labelPosition?: "bottom" | "inline";
+  /**
+   * The markers to show under the slider to label some values
+   */
+  markers?: { label: string; value: number }[];
+  /**
+   * Maximum slider value
+   */
+  max?: number;
+  /**
+   * Minimum slider value
+   */
+  min?: number;
+  /**
+   * Label for maximum value
+   */
+  maxLabel?: string;
+  /**
+   * Label for the minimum value
+   */
+  minLabel?: string;
+  /**
+   * Name of the slider input
+   */
+  name?: string;
+  /**
+   * Change handler to be used when in a controlled state
+   */
+  onChange?: (event: SyntheticEvent, value: [number, number]) => void;
+  /**
+   * Minimum interval the slider thumb can move
+   */
+  step?: number;
+  /**
+   * The multiplier for the step when using PageUp and PageDown keys via keyboard interaction
```

Comment:
`using PageUp and PageDown keys via keyboard interaction`

these repeat themselves perhaps?

could be

`The multiplier for the step when using PageUp and PageDown keys`

Reply:
equally, if you compare this to step description, we could say 
`Maximum interval the slider thumb can move when using PageUp and PageDown keys`